bl-kernel/abstract/dbjson.class.php

```diff
@@ -93,7 +93,7 @@ public function save()
 	// Returns a JSON encoded string on success or FALSE on failure.
 	private function serialize($data)
 	{
-		return json_encode($data, JSON_PRETTY_PRINT);
+		return json_encode($data, JSON_UNESCAPED_UNICODE |  JSON_PRETTY_PRINT);
```

Comment:
is necessary defined this variable for PHP 5.3, because was implemented in PHP 5.4